src/client/app/helpers/theme.ts

```diff
@@ -51,6 +51,14 @@ export function applyTheme(host: string): Promise<boolean> {
                 });
                 break;
 
+            case HostType.OUTLOOK:
+                (require as any).ensure([], (require) => {
+                    require(['style-loader!raw-loader!postcss-loader!sass-loader!../../assets/styles/themes/outlook.scss'], () => {
```

Comment:
Oh, 'cmon.  Please refactor this logic to just have it be:

    let hostStyleFile = `${(host || 'web').toLowerCase()}.scss';
    (require as any).ensure([], (require) => {
        require([`style-loader!raw-loader!postcss-loader!sass-loader!../../assets/styles/themes/${hostStyleFile}`], () => {
            resolve(true);
        });
    });